.github/workflows/rw-backup-auzre-maria-database.yml

```
@@ -0,0 +1,105 @@
+name: RW — Backup a given database within an Azure MariaDatabase' server
+
+on:
+  workflow_call:
+    inputs:
+      firewall_rule_name:
+        type: string
+        required: true
+        
+      github_env:
+        type: string
+
+      database_resource_group:
+        type: string
+        required: true
+
+      database_server_name:
+        type: string
+        required: true
+
+      database_name:
+        type: string
+        required: true
+
+      database_user:
+        type: string
+        required: true
+
+      keyvault:
+        type: string
+        required: true
+
+      keyvault_has_acl:
+        type: boolean
+        required: false
+        default: false
+
+      database_password_kv_ref:
+        type: string
+        required: true
+
+      storage_account_name:
+        type: string
+        required: true
+
+      storage_container_name:
+        type: string
+        required: true
+
+    secrets:
+      az_creds:
+        required: true
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  create_metadata:
+    runs-on: ubuntu-latest
+    outputs:
+      backup_filename: ${{ steps.metadata.outputs.backup_filename }}
+    steps:
+      - name: Create
+        id: metadata
+        run: |
+          echo "backup_filename=${{ inputs.database_server_name }}/${{ inputs.database_name}}/${{ inputs.database_server_name }}_${{ inputs.database_name}}_$(date +%Y-%m-%d_%H:%M:%S).sql" >> $GITHUB_OUTPUT
```

```suggestion
          echo "backup_filename=${{ inputs.database_server_name }}/${{ inputs.database_name}}/${{ inputs.database_server_name }}_${{ inputs.database_name}}_$(date +%Y-%m-%d_%H:%M:%S).bak" >> $GITHUB_OUTPUT
```

As its no longer a sql file (has been gzipped) we should either call it `bak` or something else like `zip`